app/assets/javascripts/discourse-common/lib/icon-library.js

```diff
@@ -1,6 +1,7 @@
 import { h } from "virtual-dom";
 import attributeHook from "discourse-common/lib/attribute-hook";
 import deprecated from "discourse-common/lib/deprecated";
+import jQuery from 'jquery';
```

Comment:
I think this should satisfy Prettier (double instead of single quotes): 
```suggestion
import jQuery from "jquery";
```